src/open_inwoner/accounts/tests/test_auth_2fa_sms.py

```diff
@@ -255,23 +211,19 @@ def test_login_token_with_max_attempts(self, mock_gateway_send):
 
             sent_token = mock_gateway_send.call_args[1]["token"]
 
-            response = self.client.post(
-                response._request.get_full_path(),
-                follow=True,
-                data={
-                    "token": sent_token,
-                },
-            )
-            other_user.refresh_from_db()
+            verify_token_form = response.forms["verify-token-form"]
+            verify_token_form["token"] = sent_token
+            response = verify_token_form.submit()
+
             self.assertRedirects(response, reverse("pages-root"))
 
         # make sure the first user can login after a minute has passed
         with freeze_time("2023-05-22 12:01:01"):
             response = self.client.post(
```

Comment:
I am using `self.client` because I cannot inject the CSRF token in the post request when I am using webtest. 
That's why we have used in the past and in another PR now, the class attribute `csrf_checks = False`

https://github.com/django-webtest/django-webtest/blob/master/README.rst